Sources/SnowManLogicSpecs/guessManagerSpec.swift

```diff
@@ -0,0 +1,163 @@
+                    		expect(guessManager.calculateRemainingGuesses()).to.equal(-1)
+                	}
+            	}
+
+            	describe("#isGuessingFullWord") {
+                	it("should return true if guess is more than 1 character") {
+                        guessManager.guess = "hello"
+                    		expect(guessManager.isGuessingFullWord()).to.equal(true)
+                	}
+
+                	it("should return false if guess is only 1 character") {
+                    		guessManager.guess = "h"
+                    		expect(guessManager.isGuessingFullWord()).to.equal(false)   
+                	}
+
+                  	it("should return false if guess is 0 characters") {
+                      		guessManager.guess = ""
+                      		expect(guessManager.isGuessingFullWord()).to.equal(false)
+                  	}
+            	}
+
+            	describe("#correctlyGuessedFullWord") {
+                	var word: String!
+                	before() {
+                    	word = "apple"
+                	}
+
+                	it("should return true if full word is guessed") {
+                    		guessManager.guess = "apple"
+                    		expect(guessManager.correctlyGuessedFullWord(word)).to.equal(true)
+                	}
+
+                	it("should return false if only part of the word is guessed") {
+                    		guessManager.guess = "app"
+                    		expect(guessManager.correctlyGuessedFullWord(word)).to.equal(false)
+                      	}
+
+                    	it("should return false if the guess is incorrect") {
+                    		guessManager.guess = "banana"
+                    		expect(guessManager.correctlyGuessedFullWord(word)).to.equal(false)
+                	}
+            	}
+
+                describe("#appendWordPlaceholder") {
+			var word: String!
+			before() {
+                        	word = "apple"
+                    	}
+
+                    	it("should add a placeholder to correctGuesses collection if the word is guessed") {
+                        	guessManager.guess = "apple"
+                        	guessManager.appendWordPlaceholder(word)
+                        	expect(guessManager.correctGuesses).to.contain("#")
+                  	}
+
+                  	it("should not add a placeholder to incorrectGuesses collection the word is guessed") {
+                        	guessManager.guess = "apple"
+                        	guessManager.appendWordPlaceholder(word)
+                        	expect(guessManager.incorrectGuesses).notTo.contain(guessManager.guess)
+                  	}
+
+                  	it("should add a placeholder to incorrectGuesses collection if the word is not guessed") {
+                        	guessManager.guess = "banana"
+                        	guessManager.appendWordPlaceholder(word)
+                        	expect(guessManager.incorrectGuesses).to.contain("#")
```

Comment:
Maybe assign the `#` to a variable in the test just to call out the fact that it's a placeholder value.